tests/testthat/test-listings.R

```diff
@@ -124,11 +124,58 @@ testthat::test_that("regression test for keycols being lost due to `head()`", {
   ## causing #41
   rlst <- as_listing(mtcars,
     key_cols = c("gear", "carb"),
-    cols = c("gear", "carb", "qsec")
+    disp_cols = "qsec"
   )
 
   testthat::expect_identical(
     dim(head(rlst, 5)),
     c(5L, ncol(mtcars))
   )
 })
+
+
+testthat::test_that("column inclusion and ordering stuff", {
+  ## key columns must be left-most k columns (#36)
+  lsting <- as_listing(
+      df = ex_adae,
+      disp_cols = c("USUBJID", "AGE", "SEX", "RACE", "ARM", "AEDECOD", "AESEV", "AETOXGR"),
+      key_cols = c("ARM", "USUBJID", "AETOXGR")
```

Comment:
order of the two , 
key_cols, then disp_cols

Reply:
these are named arguments so I don't really get why this would matter.

Reply:
Hi Gabe, just to keep coding style more consistent. and you have used key_cols, then disp_cols in other places in this PR as well. also it is consistent with the function call

Reply:
Right but both work, and this is in a test anyway. This kind of stringency doesn't really add any value (also, we support end users putting them in any order so even though that bit is essentially impossible to break, as it is a core feature of how R works, explicitly avoiding that in tests wouldn't really be the right thing to do)

Reply:
Dear @gmbecker , please accpet these changes, these small changes look very minor, but built up technical debt for the future programmers to learn, adapt and maintain. Thanks for your understanding

Reply:
Again, this is not something we should be paying attention to. Anyone who doesn't understand how named arguments work in R should not be touching code, so this is not a valuable use of anyone's time

Reply:
Hi @gmbecker , appreciate the feedback. Can you please fix the conflict, and lintr issues. I will approve this PR. Thanks a lot.